src/nvim/testdir/test_substitute.vim

```diff
@@ -765,4 +765,20 @@ func Test_substitute_skipped_range()
   bwipe!
 endfunc
 
+" Test for command failures when the last substitute pattern is not set.
+func Test_sub_with_no_last_pat()
+  " TODO how to override last substitution pattern?
```

Comment:
Thoughts here?

Reply:
Ah anyhow, they later remove it https://github.com/vim/vim/commit/4f5776c17cd86f904a7e2f92db297c73e28939b7 - so I'll port up related patches up to that one.